pkgs/applications/misc/calcoo/default.nix

```diff
@@ -39,20 +38,21 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
 
-    mkdir -p $out/bin $out/share/${pname}
-    mv dist/lib/calcoo.jar $out/share/${pname}
+    install -Dm644 dist/lib/calcoo.jar -t $out/share/calcoo
 
-    makeWrapper ${jdk}/bin/java $out/bin/calcoo \
-    --add-flags "-jar $out/share/${pname}/calcoo.jar"
+    makeWrapper ${jre}/bin/java $out/bin/calcoo \
```

Comment:
As far as I remember, JRE is basically a stripped down JDK.
It makes any difference to use `jdk` here instead?

Reply:
`jre` is an alias for the latest `jdk` by default
It turns out that only `jdk8` has an actual `jre8`

The separation can make sense in very specific cases, for example you could build the package using `jdk = jdk8` and run it with `jre = jdk11` (if you wanted to do that for some reason)

I will revert this for now, because this really is just making it a bit more complex than it needs to be